assets/app/views/builds.html

```diff
@@ -95,7 +95,7 @@
                       <span>- {{build.metadata.creationTimestamp | date : 'short'}}</span>
                     </td>
                     <td data-title="Type">{{build.spec.strategy.type}}</td>
-                    <td data-title="Source">
+                    <td data-title="Source" class="word-break-all">
```

Comment:
Yes, if the source url is long it will cause FF and IE to extend the table beyond it's container.